test_api.py

```diff
@@ -2690,82 +2692,136 @@ def test_get_only_stale_warning(self):
         retrieved_host_ids = self._get_all_hosts("?staleness=stale_warning")
         self.assertEqual((self.stale_warning_host["id"],), retrieved_host_ids)
 
+
+class QueryStalenessGetHostsIgnoresCulledTestCase(QueryStalenessGetHostsBaseCase, DeleteHostsBaseTestCase):
+    def _get_created_hosts_by_id(self, endpoint="", query=""):
+        hosts = self._get_hosts_by_id(self._created_hosts(), endpoint, query)
+        if endpoint == "/tags" or endpoint == "/tags/count":
+            print(hosts)
+            return tuple(hosts.keys())
+        else:
+            return tuple(host["id"] for host in hosts)
+
     def test_dont_get_only_culled(self):
         for get_hosts_url_method in (self._get_all_hosts_url, self._get_created_hosts_by_id_url):
             with self.subTest(get_hosts_url_method=get_hosts_url_method):
-                url = get_hosts_url_method("?staleness=culled")
+                url = get_hosts_url_method(query="?staleness=culled")
                 self.get(url, 400)
 
+    def test_get_hosts_by_id_default_ignores_culled(self):
+        retrieved_host_ids = self._get_created_hosts_by_id("")
+        expected_host_ids = (self.stale_warning_host["id"], self.stale_host["id"], self.fresh_host["id"])
+        self.assertEqual(expected_host_ids, retrieved_host_ids)
 
-class QueryStalenessConfigTimestampsTestCase(QueryStalenessBaseTestCase):
-    def _create_and_get_host(self, stale_timestamp):
-        host_to_create = self._create_host(stale_timestamp)
-        query = "?staleness=fresh,stale,stale_warning,unknown"
-        retrieved_host = self._get_hosts_by_id(query, (host_to_create["id"],))[0]
-        self.assertEqual(stale_timestamp.isoformat(), retrieved_host["stale_timestamp"])
-        return retrieved_host
+    def test_tags_default_ignores_culled(self):
+        retrieved_host_ids = self._get_created_hosts_by_id("/tags")
+        expected_host_ids = (self.stale_warning_host["id"], self.stale_host["id"], self.fresh_host["id"])
+        self.assertEqual(expected_host_ids, retrieved_host_ids)
+
+    def test_tags_count_default_ignores_culled(self):
+        retrieved_host_ids = self._get_created_hosts_by_id("/tags/count")
+        expected_host_ids = (self.stale_warning_host["id"], self.stale_host["id"], self.fresh_host["id"])
+        self.assertEqual(expected_host_ids, retrieved_host_ids)
+
+    def test_get_system_profile_ignores_culled(self):
+        retrieved_host_ids = self._get_created_hosts_by_id("/system_profile")
+        expected_host_ids = (self.stale_warning_host["id"], self.stale_host["id"], self.fresh_host["id"])
+
+        self.assertEqual(expected_host_ids, retrieved_host_ids)
+
+    def test_patch_ignores_culled(self):
+        url = HOST_URL + "/" + self.culled_host["id"]
+
+        self.patch(url, {"display_name": "patched"}, 404)
+
+    def test_patch_works_on_non_culled(self):
+        host_id_url_list = ",".join([self.fresh_host["id"]])
+        url = HOST_URL + "/" + host_id_url_list
```

Comment:
You’re joining a hard-coded list with just one item. You can just go without the joining and use the value directly.

```suggestion
        url = HOST_URL + "/" + self.fresh_host["id"]
```